athina/steps/conditional.py

```
@@ -2,29 +2,40 @@
 from athina.steps.base import Step
 from jinja2 import Environment, UndefinedError
 from pydantic import ConfigDict
+import ast
```

Ruff (0.8.2)

`ast` imported but unused

Remove unused import: `ast`

(F401)